environment-test.yml

```diff
@@ -28,6 +29,3 @@ dependencies:
   - eccodes # required by cfgrib
   - geos # required by cartopy
   - proj # required by cartopy
-
-  - pip:
-      - herbie-data
```

Comment:
I've used the `environment-test.yml` file to setup the Conda environment, because it was already close to what was needed there. I see that you might have been using it differently, because it installed the `herbie-data` package from PyPI. However, on CI, when running code from a pull request, the code from the repository should be used.

Let me know if you can adjust your workflow, or if you have other suggestions.